docker/Makefile

```diff
@@ -1,5 +1,14 @@
+# Variables can be overridden by setting environment variables
+
+FACTORIO_PATH ?= ~/.factorio/
+SECURITY_PATH ?= $(FACTORIO_PATH)/security
+SAVES_PATH ?= $(FACTORIO_PATH)/saves
+MODS_PATH ?= $(FACTORIO_PATH)/mods
+ADMIN_PASS ?= jqkSnQS4rA
```

Comment:
@drahti 

I agree, since your Dockerfile generates a random password anyway its much more secure to just leave it blank by default.